src/main/java/com/marklogic/appdeployer/command/IncrementalFilenameFilter.java

```diff
@@ -0,0 +1,12 @@
+package com.marklogic.appdeployer.command;
+
+import java.io.FilenameFilter;
+
+public interface IncrementalFilenameFilter extends FilenameFilter {
+
+	public void addFilenameToIgnoreHash(String filename);
+
+	public void clearFilenamesToIgnoreHash();
+
+	public void setIncrementalMode(boolean incrementalMode);
```

Comment:
I think this is the only method needed. The client needs to tell the filter whether or not to enable incremental support. How about "enableIncremental()" and "disableIncremental()" to make it explicit so that a boolean doesn't have to be interpreted?